### PR TITLE
Implement means for posting validator exits

### PR DESCRIPTION
**Pull request by zah**
Remaining items:

* [ ] Implement a CLI command from producing a `SignedValidatorExit` message in JSON form.
* [ ] Add a script that creates a signed message and then posts it with `curl` against a running node

### CODE REVIEW

**Comment by arnetheduck** [comment]
so far so good..

**Comment by arnetheduck** [comment]
[ ] add a page in documentation about it

**Review by arnetheduck — APPROVED** [review]

File: beacon_chain/nimbus_beacon_node.nim [hunk 1]
```diff
@@ -1025,6 +1026,143 @@ when hasPrompt:
...
+
+  try:
+    echoP "PLEASE BEWARE!"
+
+    echoP "Publishing a voluntary exit is an irreversible operation! " &
+          "You won't be able to restart again with the same validator."
+
+    echoP "By requesting an exit now, you'll be exempt from penalties " &
+          "stemming from not performing your validator duties, but you " &
+          "won't be able to withdraw your deposited funds at the time " &
```
Comment (sachayves) [inline beacon_chain/nimbus_beacon_node.nim:1123]:
```suggestion
          "won't be able to withdraw your deposited funds for the time " &
```

File: beacon_chain/nimbus_beacon_node.nim [hunk 2]
```diff
@@ -1025,6 +1026,143 @@ when hasPrompt:
...
+          "being. This means that your funds will be effectively frozen " &
+          "until withdrawals are enabled in a future phase of the Eth2 " &
```
Comment (sachayves) [inline beacon_chain/nimbus_beacon_node.nim:1125]:
```suggestion
          "until withdrawals are enabled in a future phase of Eth2."
```

File: beacon_chain/nimbus_beacon_node.nim [hunk 3]
```diff
@@ -1025,6 +1026,143 @@ when hasPrompt:
...
+          "rollout."
```
Comment (sachayves) [inline beacon_chain/nimbus_beacon_node.nim:1126]:
```suggestion
          
```

File: beacon_chain/nimbus_beacon_node.nim [hunk 4]
```diff
@@ -1025,6 +1026,143 @@ when hasPrompt:
...
+
+    echoP "To understand more about the Eth2 roadmap, we recommend you " &
+          "have a look at\n" &
+          "https://ethereum.org/en/eth2/#roadmap"
+
+    echoP "You must not shut down your validator for at least 5 epochs " &
```
Comment (sachayves) [inline beacon_chain/nimbus_beacon_node.nim:1132]:
```suggestion
    echoP "You must keep your validator running for at least 5 epochs " &
```

**Pull request merged**